routes/index.js

```diff
@@ -266,6 +266,32 @@ router.get('/changeStatus/:userid/:stat', isLoggedIn, function(req, res) {
   })
 })
 
+router.get('/check-in', isLoggedIn, function (req, res, next) {
+  if (!adminFunc.isAdmin(req.user.local.email)) {
```

Comment:
we should break out this method into a isAdmin middleware (like isLoggedIn but for admins only